src/cart/crt_hg_proc.c

```diff
@@ -400,8 +400,10 @@ crt_hg_unpack_header(hg_handle_t handle, struct crt_rpc_priv *rpc_priv,
 				    clock_offset,
 				    rpc_priv->crp_req_hdr.cch_src_rank);
 		/* Fail all but SWIM requests. */
-		if (!crt_opc_is_swim(rpc_priv->crp_req_hdr.cch_opc))
+		if (!crt_opc_is_swim(rpc_priv->crp_req_hdr.cch_opc)) {
+			crt_trigger_hlc_error_cb();
```

Comment:
This call needs the rate limiting offered by REPORT_HLC_SYNC_ERR (whose rate limiting period can be changed if you consider it too long), otherwise there may be a flood of this call if the current process's clock is behind all others too much and if the current process communicates a lot. Also, this call needs to be made for SWIM requests too.

Reply:
done, moved into REPROT_HLC_SYNC_ERR() itself